modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseUser.js

```diff
@@ -129,9 +128,7 @@ const BaseUser = ({
 						value={search}
 					/>
 
-					<ClayAutocomplete.DropDown
-						active={(!!resource && !!search) || initialLoading}
```

Comment:
I removed the initialLoading constant from the check because when the component was created, in its lifecycle, it allowed the bug to happen.
She was apparently responsible only for this check so I removed the constant from the code.